tests/lifecycle/eventhub_cases_test.go

```diff
@@ -14,5 +22,43 @@ var eventhubsTestCases = []serviceLifecycleTestCase{
 				"latest-operation": "provision",
 			},
 		},
+		testCredentials: testEventhubCreds,
 	},
 }
+
+func testEventhubCreds(credentials map[string]interface{}) error {
+	hubName := credentials["eventHubName"].(string)
+	connStrP1 := credentials["connectionString"].(string)
+	connStr := connStrP1 + ";EntityPath=" + hubName
+	hub, err := eventhub.NewHubFromConnectionString(connStr)
+	if err != nil {
+		return fmt.Errorf("error creating eventhub client object: %s", err)
+	}
+	ctx, cancel := context.WithTimeout(context.Background(), 80*time.Second)
+	defer cancel()
+	err = hub.Send(ctx, eventhub.NewEventFromString("hello, world!"))
+	if err != nil {
+		return fmt.Errorf("error sending message to eventhub: %s", err)
```

Comment:
The latest lifecycle test failed here:
```
--- FAIL: TestServices (3449.00s)
    --- FAIL: TestServices/lifecycle (0.00s)
        --- FAIL: TestServices/lifecycle/TestServices/lifecycle/eventhubs/eventhubs (63.60s)
	Error Trace:	driver_test.go:50
	Error:      	Expected nil, but got: &errors.errorString{s:"error sending message to eventhub: amqp: receiver settlement mode \"second\" requested, received \"first\" from server"}
```

And a similar issue in the client repo: https://github.com/Azure/azure-event-hubs-go/issues/101

Reply:
Is it possible to run the lifecycle tests for EH in the eastus datacenter? I created a simple send/receive function and it works on eastus but gets the same error in southcentral.

Reply:
This error should be fixed in https://github.com/Azure/azure-event-hubs-go/pull/104.

Please open an issue in EventHubs if you run into this issue again.

Reply:
Thanks @devigned for the fix! @aljannuzzi it looks changing datacenter works, too. I'm going to merge this PR. Thanks for adding the test case!